cypress/support/fragments/settings/remote-storage/configurations.js

```diff
@@ -48,35 +59,66 @@ function saveAndCloseForm() {
   cy.wait(2000);
 }
 
+function openCreateConfigurationForm() {
+  cy.do(newButton.click());
+}
+
 const configurations = {
   DematicEMS: {
     title: 'Dematic EMS',
     create(name) {
+      openCreateConfigurationForm();
       fillGeneralInfo(name, this.title);
       saveAndCloseForm();
     },
   },
   DematicStagingDirector: {
     title: 'Dematic StagingDirector',
     create(name) {
-      fillGeneralInfo(name, this.title);
-      cy.do(configurationFields.timingInput.fillIn('1'));
+      openCreateConfigurationForm();
+      this.fillRequiredFields(name);
       saveAndCloseForm();
     },
+    fillRequiredFields(name, timing = '1') {
+      fillGeneralInfo(name, this.title);
+      cy.do(configurationFields.timingInput.fillIn(timing));
+    },
+    verifyRequiredFields(name, timing = '1') {
+      cy.expect(configurationFields.nameInput.has({ value: name }));
+      cy.expect(configurationFields.timingInput.has({ value: timing }));
```

Comment:
if multiple assertions in a row, better to pass as an array, like so: 
      cy.expect([
        configurationFields.nameInput.has({ value: name }),
        configurationFields.timingInput.has({ value: timing })
      ]);

Reply:
fixed